modules/mysql/failover_replica.tf

```diff
@@ -17,7 +17,7 @@
 resource "google_sql_database_instance" "failover-replica" {
   count                 = "${var.failover_replica ? 1 : 0}"
   project               = "${var.project_id}"
-  name                  = "${var.name}-failover"
+  name                  = "${var.name}-failover${var.failover_replica_name_suffix=="" ? "" : "-"}${var.failover_replica_name_suffix}"
```

Comment:
Could we just use failover_replica_name_suffix without prepending the hyphen?

Reply:
Either way works - I had the bit prepending the hyphen to approximate the way the name had been generated previously to then produce names of the form:
`dbname-failover-suffix`

Simplifying the code to be:
```
name                  = "${var.name}-failover${var.failover_replica_name_suffix}"
```
would result in a name like:
`dbname-failoversuffix`

If a hyphen is then really wanted/needed it can be supplied as part of the suffix itself.

Reply:
I was thinking prepending hyphen is good for consistency, which has hyphen as delimiters for each word. But I don't have any strong opinions on this, so removed: https://github.com/GoogleCloudPlatform/terraform-google-sql-db/pull/46/commits/a5765c96e3473e4a8a1db09723852abbe0af0d5c